<launch>
  <node name="adc_scope" pkg="io_adc" type="adc_scope" output="screen" cwd="node" args="">
    <param name="pause_micro_seconds" value="1000"/>
    <param name="unipolar" value="true"/>
    <param name="adc_mode" type="string" value="localGND"/>
    <param name="voltage_range_0" type="string" value="5"/>
    <param name="voltage_range_1" type="string" value="5"/>
    <param name="voltage_range_2" type="string" value="5"/>
    <param name="voltage_range_3" type="string" value="5"/>
    <param name="voltage_range_4" type="string" value="5"/>
    <param name="voltage_range_5" type="string" value="5"/>
    <param name="voltage_range_6" type="string" value="5"/>
    <param name="voltage_range_7" type="string" value="5"/>
    <param name="voltage_range_8" type="string" value="5"/>
    <param name="voltage_range_9" type="string" value="5"/>
    <param name="voltage_range_10" type="string" value="5"/>
    <param name="voltage_range_11" type="string" value="5"/>
    <param name="voltage_range_12" type="string" value="5"/>
    <param name="voltage_range_13" type="string" value="5"/>
    <param name="voltage_range_14" type="string" value="5"/>
    <param name="voltage_range_15" type="string" value="5"/>
    <param name="voltage_range_16" type="string" value="5"/>
    <param name="voltage_range_17" type="string" value="5"/>
    <param name="voltage_range_18" type="string" value="5"/>
    <param name="voltage_range_19" type="string" value="5"/>
    <param name="voltage_range_20" type="string" value="5"/>
    <param name="voltage_range_21" type="string" value="5"/>
    <param name="voltage_range_22" type="string" value="5"/>
    <param name="voltage_range_23" type="string" value="5"/>
    <param name="voltage_range_24" type="string" value="5"/>
    <param name="voltage_range_25" type="string" value="5"/>
    <param name="voltage_range_26" type="string" value="5"/>
    <param name="voltage_range_27" type="string" value="5"/>
    <param name="voltage_range_28" type="string" value="5"/>
    <param name="voltage_range_29" type="string" value="5"/>
    <param name="voltage_range_30" type="string" value="5"/>
    <param name="voltage_range_31" type="string" value="5"/>
    <param name="voltage_range_32" type="string" value="5"/>
    <param name="voltage_range_33" type="string" value="5"/>
    <param name="voltage_range_34" type="string" value="5"/>
    <param name="voltage_range_35" type="string" value="5"/>
    <param name="voltage_range_36" type="string" value="5"/>
    <param name="voltage_range_37" type="string" value="5"/>
    <param name="voltage_range_38" type="string" value="5"/>
    <param name="voltage_range_39" type="string" value="5"/>
    <param name="voltage_range_40" type="string" value="5"/>
    <param name="voltage_range_41" type="string" value="5"/>
    <param name="voltage_range_42" type="string" value="5"/>
    <param name="voltage_range_43" type="string" value="5"/>
    <param name="voltage_range_44" type="string" value="5"/>
    <param name="voltage_range_45" type="string" value="5"/>
    <param name="voltage_range_46" type="string" value="5"/>
    <param name="voltage_range_47" type="string" value="5"/>
    <param name="voltage_range_48" type="string" value="5"/>
    <param name="voltage_range_49" type="string" value="5"/>
    <param name="voltage_range_50" type="string" value="5"/>
    <param name="voltage_range_51" type="string" value="5"/>
    <param name="voltage_range_52" type="string" value="5"/>
    <param name="voltage_range_53" type="string" value="5"/>
    <param name="voltage_range_54" type="string" value="5"/>
    <param name="voltage_range_55" type="string" value="5"/>
    <param name="voltage_range_56" type="string" value="5"/>
    <param name="voltage_range_57" type="string" value="5"/>
    <param name="voltage_range_58" type="string" value="5"/>
    <param name="voltage_range_59" type="string" value="5"/>
    <param name="voltage_range_60" type="string" value="5"/>
    <param name="voltage_range_61" type="string" value="5"/>
    <param name="voltage_range_62" type="string" value="5"/>
    <param name="voltage_range_63" type="string" value="5"/>
  </node>
</launch>
